### DISCUSSION TITLE
I created a mastodon server and the moderation and administration settings aren't showing up, can someone help?

### DISCUSSION THREAD

**Discussion by spacevini8** [mastodon/mastodon#23199] [Q&A] [answered]
this is my preferences page:

![brave_H6RRHrMqEm](https://user-images.githubusercontent.com/52097786/213921782-67c43d99-344b-4ca5-8f5a-9a6661122a5e.png)

it should have two  settings that say "moderation" and "administration"

**Comment by mtbluedog** [comment]
How did you install? What guide did you use?

**Comment by spacevini8** [comment, marked as answer]
I solved it! I had to set the account to an owner account in the "tootcli"

**Reply by mtbluedog** [reply]
Good that it worked. For others, read https://github.com/mastodon/mastodon/discussions/18137 and check feedmmittens solution, among others.
